models/staging/stg_reviews.sql

```diff
@@ -1,4 +1,4 @@
 {{ config(materialized='view') }}
 
 SELECT *
```

Comment:
We like to list all of the field names for all staging models, but this just the Fishtown preference :)